deploy.rinkeby.json

```diff
@@ -1,6 +1,5 @@
 {
-    "dependencyContracts": {
-        "UniswapAdapter": "0xde64ba88a87f676b057ac704793adbc76768e2f3",
-        "UniswapFactory": "0xf5D915570BC477f9B8D6C0E980aA81757A3AaC36"
-    }
+    "antToDai_uniswap_dataFee": "0x706b50670c80339b6a8a0d6e3b95952d8afc2234",
```

Comment:
since we'll probably have a lot of these, might want an easy to read naming convention like `uniswapDataFeed_ANT_DAI`